wallet-connect/.eslintignore

```diff
@@ -0,0 +1,8 @@
+**/node_modules
+**/dist
+**/package-lock.json
+.DS_Store
+**/coverage
+packages/crypto
+**/*.config.js
+.log
```

Comment:
ditto

Reply:
The first 2 points have been resolved. Other wallets have also been added and have also been verified with `yarn link`. The git version method suggested by you didn't work.